packages/weather/OpenWeatherMap.py

```diff
@@ -43,32 +44,103 @@ def wrapper_load_config(string, entities):
         return func(payload)
     return wrapper_load_config
 
+
```

Comment:
Why all these newlines?

Reply:
I follow pycodestyle recommendations with `python-ide`package for atom

Reply:
Didn't know about that package. I'll have to install it. Thanks